### DISCUSSION TITLE
Protecting app with login

### DISCUSSION THREAD

**Discussion by slimshader** [lepoco/wpfui#1127] [Q&A]
Hi,

My app needs user to be logged, what is the best approach to design UI so that when app starts / user is logged out, only the login Dialog is visible on top of naviation view?

**Comment by rasyidf** [comment]
Perhaps adding content dialog will help on the simple cases,

something like this

![image](https://github.com/lepoco/wpfui/assets/28984914/1525eb34-a509-4015-a2ac-447314b1ce59)
